src/index.html

```diff
@@ -5,8 +5,293 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a class="header__nav-logo-link" href="#">
+          <img
+            src="images/moyo_1.png"
+            alt="moyo_img"
+            class="header__nav-logo">
+        </a>
+        <nav>
+          <ul class="header__nav-list">
+            <li class="header__nav-item header__nav-item--is-active">
+              <a class="is-active" href="#Apple">Apple</a>
+            </li>
+            <li class="header__nav-item"><a class="header__nav-link" href="#Samsung">Samsung</a></li>
+            <li class="header__nav-item"><a class="header__nav-link" href="#Smartphones">Smartphones</a></li>
+            <li class="header__nav-item">
+              <a
+                data-qa="nav-hover"
+                class="header__nav-link"
+                href="#Laptops & Computers"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="header__nav-item"><a class="header__nav-link" href="#Gadgets">Gadgets</a></li>
+            <li class="header__nav-item"><a class="header__nav-link" href="#Tablets">Tablets</a></li>
+            <li class="header__nav-item"><a class="header__nav-link" href="#Photo">Photo</a></li>
+            <li class="header__nav-item">
+              <a class="header__nav-link last" href="#Video">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+
+
+    <main>
+
+      <div class="prod-card-container">
+        <div
+        data-qa="card"
+        class="product-card">
+          <img
+            class="product-card__imac-image"
+            src="images/imac.jpeg"
+            alt="imac-photo"
+          >
+          <h2 class="product-card__product-description" >APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+          <p class="product-card__product-code">Product code: 195434</p>
+          <div class="product-card__feadback">
```

Comment:
```suggestion
          >
          
          <h2 class="product-card__product-description >
            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
          </h2>
          
          <p class="product-card__product-code">Product code: 195434</p>
          
          <div class="product-card__feadback">
```

Add one empty line between sibling blocks inside the same parent block for both of them.
Same for similar cases below